### ISSUE TITLE
Bundle install

### ISSUE THREAD

**Issue by m-sakthivel** [bundler/bundler#4807]
--- ERROR REPORT TEMPLATE -------------------------------------------------------
- What did you do?
  
  I ran the command `/home/sakthivel/.rvm/gems/ruby-1.8.7-head/bin/bundle install`
- What did you expect to happen?
  
  I expected Bundler to...
- What happened instead?
  
  Instead, what actually happened was...

Error details

```
ArgumentError: wrong number of arguments (5 for 4)
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/definition.rb:200:in `resolve'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/definition.rb:200:in `resolve'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/definition.rb:166:in `missing_specs'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/installer.rb:188:in `resolve_if_need'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/ui/shell.rb:81:in `silence'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/installer.rb:185:in `resolve_if_need'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/installer.rb:70:in `run'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/installer.rb:22:in `install'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/cli/install.rb:102:in `run'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/cli.rb:175:in `install'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `__send__'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/invocation.rb:126:in `invoke_command'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor.rb:359:in `dispatch'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/vendor/thor/lib/thor/base.rb:440:in `start'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/cli.rb:11:in `start'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/exe/bundle:27
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/lib/bundler/friendly_errors.rb:98:in `with_friendly_errors'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/gems/bundler-1.12.5/exe/bundle:19
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/bin/bundle:23:in `load'
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/bin/bundle:23
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/bin/ruby_executable_hooks:15
```

Environment

  Bundler   1.12.5
  Rubygems  2.0.17
  Ruby      1.8.7 (2014-01-28 patchlevel 376) [i686-linux]
  GEM_HOME  /home/sakthivel/RS/call_me/vendor/local/ruby/1.8
  GEM_PATH  
  RVM       1.27.0 (latest)
  Git       1.9.1

```
  Bundler settings
```

  disable_shared_gems
    Set for your local app (/home/sakthivel/RS/call_me/.bundle/config): true
  path
    Set for your local app (/home/sakthivel/RS/call_me/.bundle/config): "vendor/local"
  orig_gem_path
    Set via BUNDLE_ORIG_GEM_PATH: "/home/sakthivel/.rvm/gems/ruby-1.8.7-head:/home/sakthivel/.rvm/gems/ruby-1.8.7-head@global"
  orig_path
    Set via BUNDLE_ORIG_PATH: "/home/sakthivel/.rvm/gems/ruby-1.8.7-head/bin:/home/sakthivel/.rvm/gems/ruby-1.8.7-head@global/bin:/home/sakthivel/.rvm/rubies/ruby-1.8.7-head/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/home/sakthivel/.rvm/bin:/home/sakthivel/.rvm/bin"

**Comment by segiddins** [comment]
What's your gemfile, so we can try and reproduce? Thanks!

**Comment by Temiautomatic** [comment]
@segiddins I'm having the same problem - my gemfile
`source 'https://rubygems.org'
ruby '2.2.2'
gem 'rails', '4.2.6'
gem 'sass-rails', '~> 5.0'
gem 'uglifier', '>= 1.3.0'
gem 'coffee-rails', '~> 4.1.0'
gem 'jquery-rails'
gem 'jbuilder', '~> 2.0'
gem 'therubyracer', :platforms => :ruby
group :development, :test do
  gem 'byebug'
end
group :development do
  gem 'web-console', '~> 2.0'
  gem 'spring'
end
gem 'bootstrap-sass'
gem 'high_voltage'
gem 'mysql2', '~> 0.3.18'
gem 'simple_form'
group :development do
  gem 'better_errors'
  gem 'quiet_assets'
  gem 'rails_layout'
  gem 'capistrano'
  gem 'capistrano-rails-console'
  gem 'capistrano-rails'
  gem 'capistrano-bundler'
  gem 'capistrano-rvm', '~> 0.1', require: false
end
`

**Comment by JuanitoFatas** [comment]
@Temiautomatic Hi! I have no problem ran `bundle` against the Gemfile you provided in this [comment](https://github.com/bundler/bundler/issues/4807#issuecomment-235268664):

```ruby
source 'https://rubygems.org'

ruby '2.2.2'

gem 'rails', '4.2.6'
gem 'sass-rails', '~> 5.0'
gem 'uglifier', '>= 1.3.0'
gem 'coffee-rails', '~> 4.1.0'
gem 'jquery-rails'
gem 'jbuilder', '~> 2.0'
gem 'therubyracer', :platforms => :ruby

group :development, :test do
  gem 'byebug'
end

group :development do
  gem 'web-console', '~> 2.0'
  gem 'spring'
end

gem 'bootstrap-sass'
gem 'high_voltage'
gem 'mysql2', '~> 0.3.18'
gem 'simple_form'

group :development do
  gem 'better_errors'
  gem 'quiet_assets'
  gem 'rails_layout'
  gem 'capistrano'
  gem 'capistrano-rails-console'
  gem 'capistrano-rails'
  gem 'capistrano-bundler'
  gem 'capistrano-rvm', '~> 0.1', require: false
end
```

I saw your Gemfile specified Ruby 2.2.2 but backtrace using something from Ruby 1.8.7:

```
ArgumentError: wrong number of arguments (5 for 4)
  /home/sakthivel/.rvm/gems/ruby-1.8.7-head/
```

Would you try reinstall your Ruby then reinstall Bundler and try again? Thank you!

**Comment by segiddins** [comment]
This should also be fixed in 1.13.0.rc.2

**Comment by segiddins** [comment]
This has been fixed in both the 1.13 and 1.14 series for a while.
